packages/fxa-auth-server/config/index.js

```diff
@@ -1604,6 +1604,15 @@ const conf = convict({
       },
     },
   },
+  supportPanel: {
+    secretBearerToken: {
+      default: 'YOU MUST CHANGE ME',
```

Comment:
Since this *must* have a setting, it should probably be added to [the list that aborts startup](https://github.com/mozilla/fxa/blob/9589fc7906810c085ce755852f42804476dca8e5/packages/fxa-auth-server/config/index.js#L1739) if the setting is left as default. 

Also, we should add a note to the next train's deploy doc that this config change needs doing.